src/languages/en.js

```diff
@@ -247,6 +247,7 @@ export default {
         error: {
             invalidSplit: 'Split amounts do not equal total amount',
             other: 'Unexpected error, please try again later',
+            genericCreateFailureMessage: 'Unexpected error requesting money, please try again later',
```

Comment:
Why do we have a generic create failure message? Wouldn't the server provide details on what went wrong?

Reply:
It should! I guess this is more of a "Just in case" message 🤷